walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+`+MQTT_HOST+` environment variable.
+
+// TODO can this be an script it will run from CodeReady workspaces?
+// CodeReady workspace should have access to the project we are deploying to.
+. From the terminal, change project:
++
+----
+oc project data-sync-walkthrough
+----
+
+. Update the ionic-showcase-server deployment to add the `+MQTT_HOST+`
+variable:
++
+----
+oc set env dc/ionic-showcase-server MQTT_HOST="$(oc get addressspace showcase -o jsonpath='{.status.endpointStatuses[?(@.name=="messaging")].serviceHost}')"
+----
+
+. Navigate to the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"] and paste the following query/mutation into window:
++
+----
+query listTasks {
+  allTasks {
+    title,
+    description,
+    id
+  }
+}
+
+mutation createTask {
+  createTask(title: "complete the walkthrough", description: "complete the mobile walk through") {
+    title,
+    description,
+    version,
+    id
+  }
+}
+----
+
+[type=verification]
+Does the GraphQL playground allow you run and view results of the query and mutation?
+
+[type=verificationFail]
+----
+Check the logs of the `ionic-showcase-server` pod.
+It should include the string `+connected to messaging service+`.
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+----
+
+
+
+[time=5]
+== Creating a mobile app in Mobile Develope Console
+
+The Mobile Developer Console allows mobile developers to view avaiable mobile services, connect them them to their mobile apps and get the mobile app configurations.
+
+. Log in to the link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"]
+. Click *Create Mobile App* to register your mobile app.
+. Enter the following name for your mobile app:
++
+----
+showcase-app
+----
++
+After the mobile app is registered, it is displayed on the console.
+. Click on the *showcase-app* card to display the configuration screen.
+. Click the *Mobile Services* tab.
+. Click *Bind to App* for *Data Sync* in the *Unbound Services* section.
+. Click Next on Binding screen. 
+. In Parameters Section provide {route-ionic-showcase-server-host} as `Server URL`
+// TODO - we doing this mostly to add datasync
+// Does it even make sense to bind anything else now. We can have them bound in optional steps after?
+. Bind the `Mobile Metrics` and `Identity Management` services to the app by clicking on the "Bind To App" buttons. Use the default binding options, and once the bindings are completed, you will see them in the `Bound Services` section. 
+. Now go back to the `Configuration` tab again, you will see the `mobile-services.json` file is updated with information about the bound services. If you expand the row for each service, you will see the links to each of the service. Feel free to click on the links and explore the dashboard for each of the service.
+// TODO can we use config access instead of manualy getting config. 
```

Comment:
@wei-lee Do you think we can use remote config feature here (to promote it) or rely on the old good copy paste?